modules/homefinder/homefinder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * HomeFinder Module
+ *
+ * @package RE-PRO
+ */
+
+/* Exit if accessed directly. */
+if ( ! defined( 'ABSPATH' ) ) { exit; }
+
+include( 'widgets/class-homefinder-widgets.php' );
+
+// Homes for Sale Widget.
+include( 'widgets/class-homes-for-sale-widget.php' );
+
+// Open House Widget.
+include( 'widgets/class-open-house-widget.php' );
```

Comment:
__[code_sniffer]__
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->